lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -203,14 +211,28 @@ def list_params(parameters_node)
           end
         end
 
+        rest_name = parameters_node.rest&.name
```

Comment:
It's because a `rest` parameter can only be a `Prism::RestParameterNode`, so we can safely invoke `name` on it as long as it's not `nil`.

We could invoke `parameter_name` here too if you prefer the consistency.

Reply:
It's fine if it's not needed. But why adding `Prism::RestParameterNode` to it too though? Could it be included in other parameter groups?

Reply:
Yes. It can be included in a destructured required parameter and then we need to be able to fetch its name.

```ruby
def foo((a, *b)); end
```

Reply:
Good point. It caught a bug in how we were handling the naming.